service/gcs/run_unittests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex 
```

Comment:
It is intended to replace setup_test_env. At this moment, we really only have one set of unittests (gcs) to run. So, it seems be to reasonable to group them into one script.  This is to bring up the CI quality gate as soon as we could to protect the repo from bad checkin.
We will get to all other nice Travis controls later.